source/index.html

```diff
@@ -1,12 +1,157 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html  class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="img/logo-desktop-1.svg" width="202" height="58" alt="Кэт Энерджи">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--active">
+              <a href="#">
+                Главная
+              </a>
+            </li>
+            <li class="site-list__item"><a href="#">Каталог продукции</a></li>
+            <li class="site-list__item"><a href="#">Подбор программы</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <h1 class="visually-hidden">Функциональное
+        питание для котов</h1>
+        <section class="program container">
+          <h2 class="program__title">Функциональное
+            питание для котов</h2>
+            <p class="program__text">Занялся собой? Займись котом!</p>
+            <img class="program__image" src="img/index-can-mobile-1.png" width="280" height="270" alt="Cat Energy Pro курицa">
+            <a class="program__link" href="form.html">Подобрать программу</a>
+        </section>
+        <section class="categories container">
+          <h2 class="categories__header visually-hidden">индивидуальный план для кота</h2>
```

Comment:
Вряд ли у заголовка будут ещё стили, если он визуально скрыт
```suggestion
          <h2 class="visually-hidden">индивидуальный план для кота</h2>
```